### DISCUSSION TITLE
Is the Safari version being abandoned ? ? ?

### DISCUSSION THREAD

**Discussion by Jerry23011** [darkreader/darkreader#11531] [General]
Just want to confirm if the Safari version is abandoned? We literally **BOUGHT** the app on App Store.

I trusted dark reader, that's why I **BOUGHT** it.

If dark reader can't maintain the Safari version anymore, just **open source** the code and the community can help build it.

I understand there is a fee for the App Store, but looking at the number of reviews, I don't think the **MONEY WE PAID** can't cover that $99/year cost??

If dark reader decides to abandon the Safari version, please let me know so that I can switch to another extension with **BASIC SUPPORT**

**Comment by h3lp404** [comment]
i want to know ,too

**Comment by gwcali** [comment]
Definitely feels like Safari is abandoned.

**Comment by alex-kinokon** [comment]
paging @alexanderby

**Comment by MikaelFangel** [comment]
There was an update on the App Store yesterday (:

**Reply by alex-kinokon** [reply]
After a whole year 😄 looks like my ping worked

**Comment by alexanderby** [comment]
Hi everyone. The update should be available starting from today. If you have any issues during the migration, please see some notes [here](https://darkreader.org/safari/).

The settings will not be migrated, but I had to push the update anyway because of Safari 17 issues.

**Comment by derekschrock** [comment]
Is the hotkey to toggle current website something that is broken or outside of the extension's control?  Hot key for toggling the extension appears to work.

**Reply by alexanderby** [reply]
Thanks for noticing, that is a bug, I will make an update shortly.

**Reply by derekschrock** [reply]
Did this update ever hit the App Store?
